cmd/bankerase/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! ## `humility bankerase`
+//!
+//! Erase some memory please write some words
+//!
+//!
+
+use anyhow::{bail, Result};
+use clap::{CommandFactory, Parser};
+use humility::cli::Subcommand;
+use humility_cmd::{Archive, Command, CommandKind};
+
+#[derive(Parser, Debug)]
+#[clap(name = "bankerase", about = env!("CARGO_PKG_DESCRIPTION"))]
+struct FlashArgs {
+    /// force re-flashing if archive matches
+    #[clap(long, short = 'F')]
+    force: bool,
+
+    /// reset delay
+    #[clap(
+        long = "reset-delay", short = 'd',
+        default_value_t = 100, value_name = "timeout_ms",
+        parse(try_from_str = parse_int::parse)
+    )]
+    reset_delay: u64,
+
```

Comment:
Would be good to indicate what these options are for so they show up in `humility bankerase --help`

Reply:
Docs added in https://github.com/oxidecomputer/humility/pull/312/commits/ea6b8bc523789eb14f41624a2e7342ced5d65a6c